lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -4532,7 +4533,17 @@ namespace Js
     template <bool doProfile>
     Var InterpreterStackFrame::ProfiledDivide(Var aLeft, Var aRight, ScriptContext* scriptContext, ProfileId profileId)
     {
-        Var result = JavascriptMath::Divide(aLeft, aRight,scriptContext);
+        Var result;
+#ifdef _M_IX86
+        if (AutoSystemInfo::Data.SSE2Available())
```

Comment:
Why only for x86 ? Also, is it not better to move the SSE2Available check into JavascriptMath::Divide ?